packages/config/config/devices/0x010f/fgk101.json

```diff
@@ -415,158 +369,277 @@
-			"valueSize": 2,
-			"minValue": 0,
-			"maxValue": 2048,
-			"defaultValue": 0
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Scene: Opening Door",
+			"valueSize": 2
+		},
+		{
+			"#": "70[0x0002]",
+			"$if": "firmwareVersion >= 3.2",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Scene: Closing Door",
+			"valueSize": 2
+		},
+		{
+			"#": "70[0x0004]",
+			"$if": "firmwareVersion >= 3.2",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Scene: Holding",
+			"valueSize": 2
+		},
+		{
+			"#": "70[0x0008]",
+			"$if": "firmwareVersion >= 3.2",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Scene: Releasing",
+			"valueSize": 2
+		},
+		{
+			"#": "70[0x0010]",
+			"$if": "firmwareVersion >= 3.2",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Scene: Double Click",
+			"valueSize": 2
+		},
+		{
+			"#": "70[0x0020]",
+			"$if": "firmwareVersion >= 3.2",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Scene: Triple Click",
+			"valueSize": 2
+		},
+		{
+			"#": "70[0x0100]",
+			"$if": "firmwareVersion >= 3.2",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Scene: High Temperature - Door Open",
+			"valueSize": 2
+		},
+		{
+			"#": "70[0x0200]",
+			"$if": "firmwareVersion >= 3.2",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Scene: High Temperature - Door Closed",
+			"valueSize": 2
+		},
+		{
+			"#": "70[0x0400]",
+			"$if": "firmwareVersion >= 3.2",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Scene: Low Temperature - Door Open",
+			"valueSize": 2
+		},
+		{
+			"#": "70[0x0800]",
+			"$if": "firmwareVersion >= 3.2",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Scene: Low Temperature - Door Closed",
+			"valueSize": 2
+		},
+		{
+			"#": "71[0x01]",
+			"$if": "firmwareVersion >= 3.2",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Broadcast Commands",
+			"valueSize": 1
+		},
+		{
+			"#": "71[0x02]",
+			"$if": "firmwareVersion >= 3.2",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Broadcast Sensor Alarm",
+			"valueSize": 1
+		},
+		{
+			"#": "71[0x04]",
+			"$if": "firmwareVersion >= 3.2",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Broadcast Tamper Alarm",
+			"valueSize": 1
 		},
 		{
-			"#": "71",
+			"#": "72[0x01]",
 			"$if": "firmwareVersion >= 3.2",
-			"label": "Alarm broadcast",
-			"description": "Settings for broadcasting ON/OFF commands, sensor ...",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "2nd Association Group (Control) Sent as Secure",
```

Comment:
I think we have specific templates for the secure commands bitmasks, please check if you can use them (or copy the labels atleast)